front-end/src/app/reports/f3x/submission-workflow/submit-f3x-step1.component.ts

```diff
@@ -93,14 +93,24 @@ export class SubmitF3xStep1Component implements OnInit, OnDestroy {
     this.form.patchValue({
       change_of_address: this.report?.change_of_address !== null ? this.report?.change_of_address : false,
       street_1: this.report?.street_1 ? this.report.street_1 : committeeAccount?.street_1,
-      street_2: this.report?.street_2 ? this.report.street_2 : committeeAccount?.street_2,
       city: this.report?.city ? this.report.city : committeeAccount?.city,
       state: this.report?.state ? this.report.state : committeeAccount?.state,
       zip: this.report?.zip ? this.report.zip : committeeAccount?.zip,
       confirmation_email_1:
         this.report?.confirmation_email_1 !== null ? this.report?.confirmation_email_1 : committeeAccount?.email,
       confirmation_email_2: this.report?.confirmation_email_2 !== null ? this.report?.confirmation_email_2 : null,
     });
+
+    //Get `street_2` from the same place the form got `street_1`
```

Comment:
I think this can be accomplished how it was before, but just making the condition street1 rather than street2:
```
street_2: this.report?.street_1 ? this.report.street_2 : committeeAccount?.street_2,
```

Reply:
Wound up solving this by using the solution you described below for the treasurer name fields.  I feel like that's just a super-readable solution.